notebooks/official/automl/automl_tabular_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1084 @@
+{
```

Comment:
<p>Are dataflow admin, storage admin, and bigquery admin really required?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/694/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
@helinwang

Reply:
Yes. The container in MP by default runs in the GCE default service account. The container launches Dataflow job (dataflow admin), read / write GCS (storage admin), creates temp BQ table when exporting (bigquery admin).

I haven't delved into more fine grained roles, if you feel there are better ones, let me know, I can test it out.

Reply:
Yeah, I think we should test out: Dataflow Developer, Storage Object Creator, Storage Object Viewer, BigQuery Data Editor. Most customers I've worked with aren't allowed to have any admin roles in their projects.

Reply:
Okay, let me test it out. Btw, maybe we don't have to make it a blocker for this PR.

Reply:
<p>It appears that the table (|Service account email|Description|Roles|) is not being rendered properly. Notebooks don't support markdown?</p>

Reply:
I think the render issue is due to ReviewNB Github plugin. It's rendering correctly in colab / VS Code.

Michael, are you okay with I follow up with the permission roles? I tested the one you mentioned, write to GCS has some problem. I need more time to figure it out.

Reply:
<p>Yes, some things don't render well on ReviewNB.</p>